README.md

```diff
@@ -179,4 +184,25 @@ Send data to a model via REST API request from R for a prediction.
 #### Example
 ```python
 p.predict("LPOptimizer_model", {"activities": ["sleep", "work", "leisure"], "required_hours": [7, 10, 0], "happiness_per_hour": [1.5, 1, 2]})
-```
\ No newline at end of file
+```
+<hr>
+### `Promote.metadata()`
```

Comment:
This isn't rendering
![image](https://user-images.githubusercontent.com/5350074/39070629-500391fa-44a1-11e8-9998-ac17204c8b6a.png)